BGAnimations/ScreenSelectMusic overlay/Leaderboard.lua

```diff
@@ -127,18 +130,23 @@ local LeaderboardRequestProcessor = function(res, master)
 				if data[playerStr]["gsLeaderboard"] then
 					leaderboardList[#leaderboardList + 1] = {
 						Name="GrooveStats",
-						Data=DeepCopy(data[playerStr]["gsLeaderboard"])
+						Data=DeepCopy(data[playerStr]["gsLeaderboard"]),
+						IsEX=false
 					}
 					master[pn]["LeaderboardIndex"] = 1
 				end
 
-				-- Then any additional leaderboards.
-				if data[playerStr]["rpg"] and data[playerStr]["rpg"]["rpgLeaderboard"] then
-					leaderboardList[#leaderboardList + 1] = {
-						Name=data[playerStr]["rpg"]["name"],
-						Data=DeepCopy(data[playerStr]["rpg"]["rpgLeaderboard"])
-					}
-					master[pn]["LeaderboardIndex"] = 1
+				-- Then any event leaderboards.
+				local events = {"rpg", "itl"}
+				for event in ivalues(events) do
+					if data[playerStr][event] and data[playerStr][event][event.."Leaderboard"] then
+						leaderboardList[#leaderboardList + 1] = {
+							Name=(data[playerStr][event]["name"]):gsub("International Timing League", "ITL"),
```

Comment:
Can we fix the name server side? I think we shouldn't have special cases like this in the code.

Reply:
Removed